diff --git a/node_modules/@esri/calcite-components-react/dist/components.js b/node_modules/@esri/calcite-components-react/dist/components.js
index 1292beb..7d25d25 100644
--- a/node_modules/@esri/calcite-components-react/dist/components.js
+++ b/node_modules/@esri/calcite-components-react/dist/components.js
@@ -1,7 +1,7 @@
 /* eslint-disable */
 /* tslint:disable */
 /* auto-generated react proxies */
-import { createReactComponent } from './react-component-lib';
+import { createReactComponent } from './react-component-lib/index.js';
 export const CalciteAccordion = /*@__PURE__*/ createReactComponent('calcite-accordion');
 export const CalciteAccordionItem = /*@__PURE__*/ createReactComponent('calcite-accordion-item');
 export const CalciteAction = /*@__PURE__*/ createReactComponent('calcite-action');
diff --git a/node_modules/@esri/calcite-components-react/dist/index.js b/node_modules/@esri/calcite-components-react/dist/index.js
index 40b494c..b7222e0 100644
--- a/node_modules/@esri/calcite-components-react/dist/index.js
+++ b/node_modules/@esri/calcite-components-react/dist/index.js
@@ -1 +1 @@
-export * from "./components";
+export * from "./components.js";
diff --git a/node_modules/@esri/calcite-components-react/dist/react-component-lib/createComponent.js b/node_modules/@esri/calcite-components-react/dist/react-component-lib/createComponent.js
index 1380335..e85040f 100644
--- a/node_modules/@esri/calcite-components-react/dist/react-component-lib/createComponent.js
+++ b/node_modules/@esri/calcite-components-react/dist/react-component-lib/createComponent.js
@@ -10,7 +10,7 @@ var __rest = (this && this.__rest) || function (s, e) {
     return t;
 };
 import React, { createElement } from 'react';
-import { attachProps, camelToDashCase, createForwardRef, dashToPascalCase, isCoveredByReact, mergeRefs } from './utils';
+import { attachProps, camelToDashCase, createForwardRef, dashToPascalCase, isCoveredByReact, mergeRefs } from './utils/index.js';
 export const createReactComponent = (tagName, ReactComponentContext, manipulatePropsFunction, defineCustomElement) => {
     if (defineCustomElement !== undefined) {
         defineCustomElement();
diff --git a/node_modules/@esri/calcite-components-react/dist/react-component-lib/createOverlayComponent.js b/node_modules/@esri/calcite-components-react/dist/react-component-lib/createOverlayComponent.js
index 9017f4d..9d76137 100644
--- a/node_modules/@esri/calcite-components-react/dist/react-component-lib/createOverlayComponent.js
+++ b/node_modules/@esri/calcite-components-react/dist/react-component-lib/createOverlayComponent.js
@@ -20,7 +20,7 @@ var __rest = (this && this.__rest) || function (s, e) {
 };
 import React from 'react';
 import ReactDOM from 'react-dom';
-import { attachProps, dashToPascalCase, defineCustomElement, setRef } from './utils';
+import { attachProps, dashToPascalCase, defineCustomElement, setRef } from './utils/index.js';
 export const createOverlayComponent = (tagName, controller, customElement) => {
     defineCustomElement(tagName, customElement);
     const displayName = dashToPascalCase(tagName);
diff --git a/node_modules/@esri/calcite-components-react/dist/react-component-lib/index.js b/node_modules/@esri/calcite-components-react/dist/react-component-lib/index.js
index 85e81ad..611f53c 100644
--- a/node_modules/@esri/calcite-components-react/dist/react-component-lib/index.js
+++ b/node_modules/@esri/calcite-components-react/dist/react-component-lib/index.js
@@ -1,2 +1,2 @@
-export { createReactComponent } from './createComponent';
-export { createOverlayComponent } from './createOverlayComponent';
+export { createReactComponent } from './createComponent.js';
+export { createOverlayComponent } from './createOverlayComponent.js';
diff --git a/node_modules/@esri/calcite-components-react/dist/react-component-lib/utils/attachProps.js b/node_modules/@esri/calcite-components-react/dist/react-component-lib/utils/attachProps.js
index 4502a2e..62385ce 100644
--- a/node_modules/@esri/calcite-components-react/dist/react-component-lib/utils/attachProps.js
+++ b/node_modules/@esri/calcite-components-react/dist/react-component-lib/utils/attachProps.js
@@ -1,4 +1,4 @@
-import { camelToDashCase } from './case';
+import { camelToDashCase } from './case.js';
 export const attachProps = (node, newProps, oldProps = {}) => {
     // some test frameworks don't render DOM elements, so we test here to make sure we are dealing with DOM first
     if (node instanceof Element) {
diff --git a/node_modules/@esri/calcite-components-react/dist/react-component-lib/utils/index.js b/node_modules/@esri/calcite-components-react/dist/react-component-lib/utils/index.js
index 75ccc14..f16829f 100644
--- a/node_modules/@esri/calcite-components-react/dist/react-component-lib/utils/index.js
+++ b/node_modules/@esri/calcite-components-react/dist/react-component-lib/utils/index.js
@@ -27,5 +27,5 @@ export const defineCustomElement = (tagName, customElement) => {
         customElements.define(tagName, customElement);
     }
 };
-export * from './attachProps';
-export * from './case';
+export * from './attachProps.js';
+export * from './case.js';
